home/web/client/components/AccountsChooser/AccountsChooser.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import DropDownMenu from 'material-ui/DropDownMenu';
 import MenuItem from 'material-ui/MenuItem';
 
-import isEqual from 'lodash.isequal';
```

Comment:
after using [lodash babel plugin](https://github.com/lodash/babel-plugin-lodash) it works like that.
build benchmark:
ATM: 7.75 MB
with plugin: 6.53MB
with [lodash webpack plugin](https://www.npmjs.com/package/lodash-webpack-plugin): 6.23MB

adding now the commit with changes